tests/terraform/checks/module/registry/test_ModuleCheck.py

```diff
@@ -24,6 +24,8 @@ def test_module_version(self):
 
         check = next(c for c in module_registry.checks["module"] if c.id == "CKV_TF_MODULE_1")
         module_registry.checks["module"].remove(check)
+        check = next(c for c in module_registry.checks["module"] if c.id == "CKV_TF_MODULE_2")
+        module_registry.checks["module"].remove(check)
```

Comment:
1. sure, let me add a comment. It is purely for cleaning without removing any other possible checks, which don't exist for `module` checks currently, but if they are added, then it makes more sense 😄 
2. As far as I know, we don't have, because there was never a real need for it. This kind of issue only happens during unit testing, because we check registry is a singleton